### PR TITLE
Resync the chain if the ledger is missing or at height 0

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: src/blockchain_sup.erl [hunk 1]
```diff
@@ -104,8 +104,8 @@ init(Args) ->
         ?WORKER(blockchain_lock, []),
         ?WORKER(blockchain_swarm, [SwarmWorkerOpts]),
         ?WORKER(?EVT_MGR, blockchain_event, [BEventOpts]),
-        ?WORKER(blockchain_worker, [BWorkerOpts]),
```
Comment (madninja) [inline src/blockchain_sup.erl]: just moving a line around?

**Comment by Vagabond** [comment]
This actually didn't work because of the automatic ledger reset (that I just removed). Trying again.

File: src/blockchain_sup.erl (hunk 1 above)
Comment (Vagabond) [reply src/blockchain_sup.erl]: Process dependencies are hell

**Comment by Vagabond** [comment]
This seems to have worked for me, please try it.

**Comment by Vagabond** [comment]
Don't merge this as it breaks the miners consensus manager.

**Pull request merged**